rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -129,7 +129,19 @@ object RholangCLI {
     Interpreter[Coeval]
       .buildNormalizedTerm(source)
       .runAttempt
-      .fold(_.printStackTrace(Console.err), processTerm)
+      .fold(
+        {
+          // ParserError seems to be about parser construction,
+          // i.e. a design-time error that merits a stack trace.
+          // LexerError is uses that way too, but it's also
+          // used for unexpected EOF and end of string.
+          case er: LexerError      => Console.err.println(er.getMessage)
+          case er: SyntaxError     => Console.err.println(er.getMessage)
+          case er: NormalizerError => Console.err.println(er.getMessage)
+          case th                  => th.printStackTrace(Console.err)
+        },
+        processTerm
+      )
```

Comment:
This change will obscure any debug info there might be once those things happen.

Also, reasoning about how the exception is thrown at the place it's being caught is often wrong and quickly obsolete.

Granted, it should be up to the caller to decide what happens when there's an error. This mandates making processFile return an Either[Throwable, Unit] and doing the error handling on the call sites.

Please drop this commit. Feel free to do the `Either` change if it helps your goals.

Reply:
> This change will obscure any debug info there might be once those things happen.

If there's a syntax error (or file not found, or...), that's an error in the input supplied by the user. A stack trace is just going to obscure that fact.